src/software/thunderscope/gl/layers/gl_sandbox_world_layer.py

```diff
@@ -291,6 +291,13 @@ def toggle_play_state(self) -> bool:
 
         return curr_play_state
 
+    def set_simulation_speed(self, speed: float) -> None:
+        """
+        Sets the speed of the simulator
+        :param speed: the new speed to set
+        """
+        super().set_simulation_speed(speed)
+
```

Comment:
if this is extended, why do we need this implementation here?

Reply:
bad merge probably